vimrc

```diff
@@ -75,7 +75,6 @@ if executable('ag')
 endif
 
 " Color scheme
-colorscheme github
 highlight NonText guibg=#060606
```

Comment:
I think it's a good idea, but we don't know what background people are using and can't be sure this will be an affective highlight.